code/modules/html_interface/paintTool/custom_painting_datum.dm

```diff
@@ -493,6 +494,7 @@
 	painting.author = author
 	painting.description = description
 	painting.copy = 1
+	painting.show_on_scoreboard = FALSE //sorry, OC only
```

Comment:
what if I print a painting then edit it afterwards (say, printing the captain self portrait then sabotaging it with stink lines and the like during a rev round), would it still not show on scoreboard?

Reply:
likely so